pages/pipelines/secrets.md.erb

```diff
@@ -133,7 +133,7 @@ steps:
 
 Referencing secrets in your steps risks them being interpolated, uploaded to Buildkite, and shown in plain text in the "Uploaded Pipelines" list in the job's Timeline tab.
 
-The Buildkite agent does [redact strings](/docs/pipelines/managing-log-output#redacted-environment-variables) that match the values off of environment variables whose names match common password patterns such as `*_PASSWORD`, `*_SECRET` and `*_TOKEN`.
+The Buildkite agent does [redact strings](/docs/pipelines/managing-log-output#redacted-environment-variables) that match the values off of environment variables whose names match common password patterns such as `*_PASSWORD`, `*_SECRET`, `*_TOKEN`, `*_ACCESS_KEY` and `*_SECRET_KEY`.
```

Comment:
Can you also please update the values at `/docs/pipelines/managing-log-output#redacted-environment-variables`? It only lists the three that were originally here.